roles/common/defaults/main.yml

```diff
@@ -87,6 +87,9 @@ diskgroup_compatible_rdbms: "{# assign to variable c compatible parameter and se
 
 run_cluvfy: false
 
+# section for DR
```

Comment:
Is it possible to explain what standby_suffix means here?  Specifically that Oracle requires each site to have a different unique name, and that these characters are appended to the database unique name on the new standby site.  As the standby site can be switched to a primary role in the future, it's a good practice to use a site name here, such as the physical region.

Reply:
it is just some addition to be used for db_unique_name on standby and for database name in DG.
I wanted to make it simple as of now at least since naming approach can be different.
However having it as variable allows to reassign it and make based on requirements.
Based on comments below I will change it to "_s"

Reply:
I meant to say that this kind of thing is good to describe in a comment or other doc, so that a user can understand what the possible options are and the implications of doing so.  For example, Oracle's silent install templates are copiously documented :-)

Reply:
I put comments as 
"# db_unique_name for standby is formed as concatenation 
# of db_name and standby_suffix to provide uniqueness in Data Guard"

Reply:
Thanks!  I think we're missing the key point though:  that db_unique_names remain static even if roles change, so that we'd recommend that a user uses a site-specific suffix rather than a role-specific one.  (so _sydney instead of _standby)